src/Raven.Client/Json/Converters/StreamConverter.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Raven.Client.Extensions.Streams;
+using Sparrow.Json;
+
+namespace Raven.Client.Json.Converters
+{
+    internal sealed class StreamConverter : RavenJsonConverter
+    {
+        public static readonly StreamConverter Instance = new StreamConverter();
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (!(writer is BlittableJsonWriter blittableJsonWriter))
+            {
+                throw new SerializationException($"{nameof(BlittableJsonWriter)} must to be used for serialize {nameof(Stream)}");
```

Comment:
Better error specify what was the value we did get here

Reply:
done